### PR TITLE
added features for Friendly theme

### PR DESCRIPTION
**Pull request by firinael**

### CODE REVIEW

File: 02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md [hunk 1]
```diff
@@ -35,7 +35,18 @@ Modes: [[Dark-mode themes|dark]]
 
 ## Features
 
-
+Custom support for:
+
+- ITS-like asides, image captioning and positioning (no callouts because I don't understand them yet);
```
Comment (lguenth) [inline 02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md:40]: Would you be so kind to remove the semicolons and the trailing dot at the end of the list items? Again, just for style/formatting consistency.

**Review by lguenth — CHANGES_REQUESTED** [review]
Hey @firinael!
Thank you so much for contributing, the additions are great – especially because most theme pages are lacking a proper feature section.

I added two comments regarding the formatting. If you have the time, it would be great if you could make the requested changes. :orange_heart:

File: 02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md [hunk 2]
```diff
@@ -35,7 +35,18 @@ Modes: [[Dark-mode themes|dark]]
...
+- Kanban plugin;
+- Page-breaking on print with double line-break (--- in two consecutive lines);
+- Quick Explorer plugin;
+- cMenu plugin;
+- Alternative table styling (excel or excelite class in frontmatter);
+- Center and cite styling (<b class=cen>centered text</b> or <b class=cit>source</b>);
```
[the remark this answers is not in the document]
Comment (firinael) [reply 02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md]: the HTML tags should've been formatted as code, my bad. fixed.

File: 02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md (hunk 1 above)
Comment (firinael) [reply 02 - Community Expansions/02.05 All Community Expansions/Themes/Friendly.md]: removed

**Review by lguenth — APPROVED** [review]
Sorry it took me so long, this looks great. Thank you!

**Pull request merged**